angular/src/app/book-table/shared/book-table.service.spec.ts

```diff
@@ -1,5 +1,5 @@
 import { TestBed, inject } from '@angular/core/testing';
-import { HttpModule} from '@angular/http';
+import { HttpModule } from '@angular/http';
```

Comment:
doesn't this formatting changes be addressed on a separate PR to keep a clean history? do we have any politic about this?